packages/gatsby/src/utils/webpack.config.js

```diff
@@ -737,21 +752,21 @@ module.exports = async (
       },
       // TODO fix our partial hydration manifest
       mangleExports: !isPartialHydrationEnabled,
-      splitChunks,
+      // splitChunks,
```

Comment:
`splitChunks` need to be rethinkied. No function callbacks allowed here (`test`, `name`, etc)